docs/book/getting_started/installation_and_setup.md

```diff
@@ -27,11 +27,11 @@ export PATH=$PATH:/usr/local/kubebuilder/bin
 
 {% sample lang="linux" %}
 ```bash
-version=1.0.6 # latest stable version
+version=1.0.7 # latest stable version
 arch=amd64
 
 # download the release
-curl -L -O https://github.com/kubernetes-sigs/kubebuilder/releases/download/v${version}/kubebuilder_${version}_linux_${arch}.tar.gz
+curl -L -O "https://github.com/kubernetes-sigs/kubebuilder/releases/download/v${version}/kubebuilder_${version}_linux_${arch}.tar.gz"
 
 # extract the archive
 tar -zxvf kubebuilder_${version}_linux_${arch}.tar.gz
```

Comment:
Can you also please update quick_start.md ?

Reply:
fixed in cdffed0